dags/main_summary.py

```diff
@@ -219,6 +223,59 @@
     executor=GetDefaultExecutor(),
     dag=dag)
 
+main_summary_experiments_get_experiment_list = gke_command(
+    task_id="main_summary_experiments_get_experiment_list",
+    command=["python3", "templates/telemetry_derived/experiments_v1/get_experiment_list.py", "{{ds}}"],
+    docker_image="mozilla/bigquery-etl:latest",
+    xcom_push=True,
+    owner="ssuh@mozilla.com",
+    email=["telemetry-alerts@mozilla.com", "frank@mozilla.com", "ssuh@mozilla.com", "robhudson@mozilla.com"],
+    dag=dag)
+
+sql_main_summary_experiments = bigquery_etl_query(
+    task_id="sql_main_summary_experiments",
+    destination_table="experiments_v1",
+    parameters=(
+        "experiment_list:ARRAY<STRING>:{{task_instance.xcom_pull('main_summary_experiments_get_experiment_list') | tojson}}",
+    ),
+    project_id="moz-fx-data-shared-prod",
+    dataset_id="telemetry_derived",
+    owner="ssuh@mozilla.com",
+    email=["telemetry-alerts@mozilla.com", "frank@mozilla.com", "ssuh@mozilla.com", "robhudson@mozilla.com"],
+    dag=dag)
+
+main_summary_experiments_export = SubDagOperator(
+    subdag=export_to_parquet(
+        table="moz-fx-data-shared-prod:telemetry_derived.experiments_v1${{ds_nodash}}",
+        destination_table="sql_experiments_v1",  # TODO remove this line when removing the databricks job
+        arguments=[
+            "--partition-by",
+            "experiment_id",
+            "submission_date_s3",
+            "--select",
+            "*",
+            "'{{ds_nodash}}' AS submission_date_s3",
+            "--replace",
+            "'{{ds_nodash}}' AS submission_date",
+            "SAFE_CAST(sample_id AS INT64) AS sample_id",
```

Comment:
Same here re: `--replace`?

Reply:
`--replace` uses `nargs="+"`, so we specify the flag once, and 1 or more values